dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -274,9 +270,20 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
                     THROW_HR_IF(E_INVALIDARG, s_protocolRegistration);
                 }
 
+                s_taskClsid = details.TaskClsid();
                 wil::unique_cotaskmem_string unpackagedAppUserModelId;
                 RegisterUnpackagedApplicationHelper(GUID_NULL, unpackagedAppUserModelId); // create default registration for app
```

Comment:
Add a comment here that unpackagedAppUserModelId is generated by the Long Running Process